nameko/web/server.py

```diff
@@ -93,6 +93,7 @@ def start(self):
     def stop(self):
         self._is_accepting = False
         self._gt.kill()
+        self._sock.close()
```

Comment:
It is the listening socket, and the greenthread doing the accepting is already dead.

Unrelated to this change though, I don't think the webserver is a good citizen with regard to letting in-flight calls finish. It doesn't track the greenthreads spawned for each request, so ``stop`` may exit while they're still running, in which case the container will kill them ungracefully. I'll raise as a separate issue.